src/lib.rs

```diff
@@ -455,8 +432,6 @@ pub fn run(input: Input, config: &Config) {
     let write_result = filemap::write_all_files(&file_map, &mut out, config);
 
     if let Err(msg) = write_result {
-        if !ignore_errors {
```

Comment:
This `if` was needed when errors were output to `stdout`. Printing to `stderr` should be always ok.